build/Codecoverage.proj

```diff
@@ -17,8 +17,8 @@
       <_BranchName Condition="'$(_BranchName)' == ''">$(BUILD_SOURCEBRANCHNAME)</_BranchName>
     </PropertyGroup>
 
-    <Message Importance="high" Text="&quot;$(_ReportGeneratorPath)&quot; -reports:$(BaseOutputPath)$(PlatformConfig)\coverage\*.coverage -targetdir:$(BaseOutputPath)$(PlatformConfig)\coverage -reporttypes:Cobertura" />
-    <Exec Command="&quot;$(_ReportGeneratorPath)&quot; -reports:$(BaseOutputPath)$(PlatformConfig)\coverage\*.coverage -targetdir:$(BaseOutputPath)$(PlatformConfig)\coverage -reporttypes:Cobertura" />
+    <Message Importance="high" Text="&quot;$(_ReportGeneratorPath)&quot; -reports:$(BaseOutputPath)$(PlatformConfig)\coverage\*.coverage -targetdir:$(BaseOutputPath)$(PlatformConfig)\coverage -filefilters:+https* -reporttypes:Cobertura" />
+    <Exec Command="&quot;$(_ReportGeneratorPath)&quot; -reports:$(BaseOutputPath)$(PlatformConfig)\coverage\*.coverage -targetdir:$(BaseOutputPath)$(PlatformConfig)\coverage -filefilters:+https* -reporttypes:Cobertura" />
```

Comment:
📝 This is an attempt to address a change in the way numbers were appearing in the new reports. For some reason, the file submitted to codecov had two copies of each source file, line this:

```xml
<class name="Microsoft.ML.StaticPipe.LdaFitResult" filename="D:\a\1\s\src\Microsoft.ML.StaticPipe\LdaStaticExtensions.cs" line-rate="0" branch-rate="1" complexity="NaN">
  <methods>
    <method name=".ctor" signature="(Microsoft.ML.Transforms.Text.LatentDirichletAllocationTransformer/LdaSummary)" line-rate="0" branch-rate="1">
      <line number="24" hits="0" branch="false" />
      <line number="25" hits="0" branch="false" />
      <line number="26" hits="0" branch="false" />
      <line number="27" hits="0" branch="false" />
    </method>
  </methods>
  <lines>
    <line number="24" hits="0" branch="false" />
    <line number="25" hits="0" branch="false" />
    <line number="26" hits="0" branch="false" />
    <line number="27" hits="0" branch="false" />
  </lines>
</class>
<class name="Microsoft.ML.StaticPipe.LdaFitResult" filename="https://raw.githubusercontent.com/dotnet/machinelearning/d495bb4c7a58c459e753b9de3d0895c188468b58/src/Microsoft.ML.StaticPipe/LdaStaticExtensions.cs" line-rate="1" branch-rate="1" complexity="NaN">
  <methods>
    <method name=".ctor" signature="(Microsoft.ML.Transforms.Text.LatentDirichletAllocationTransformer/LdaSummary)" line-rate="1" branch-rate="1">
      <line number="24" hits="1" branch="false" />
      <line number="25" hits="1" branch="false" />
      <line number="26" hits="1" branch="false" />
      <line number="27" hits="1" branch="false" />
    </method>
  </methods>
  <lines>
    <line number="24" hits="1" branch="false" />
    <line number="25" hits="1" branch="false" />
    <line number="26" hits="1" branch="false" />
    <line number="27" hits="1" branch="false" />
  </lines>
</class>
```